pyam/iiasa.py

```diff
@@ -333,15 +343,15 @@ def query(self, **kwargs):
             df[META_IDX + ['version']].drop_duplicates()
             .groupby(META_IDX).count().version
         )
-        if max(lst) > 1:
+        if not lst.empty and max(lst) > 1:
```

Comment:
can't this be done with `if len(lst) > 1`?

Reply:
Sure. Just out of curiosity - why it is better in your opinion in this particular case?

Reply:
Ah, I think I misunderstood this. You're checking if the version is `>1`? Then your previous implementation was correct - my apologies. It would be worth adding a comment saying that version has to be `>1` and why.